src/test/java/org/jreliability/sl/SLTest.java

```diff
@@ -31,7 +31,7 @@
 /**
  * The The {@link SLTestTest} tests the {@link SL}.
  * 
- * @author glass, jlee
+ * @author glass, ilee

```

Comment:
ailee?

Reply:
jlee is correct, seems I made a refactor mistake (also in the The The)